src/test/groovy/graphql/execution/instrumentation/dataloader/PeopleCompaniesAndProductsDataLoaderTest.groovy

```diff
@@ -92,13 +92,12 @@ class PeopleCompaniesAndProductsDataLoaderTest extends Specification {
     DataFetcher productsDF = new DataFetcher() {
         @Override
         Object get(DataFetchingEnvironment environment) {
-            return IntStream.range(0, 5)
```

Comment:
There is some groovy bugs where streams dont work on Java 11 - I am not 100% sure of the reasoning but re-writiing makes it work